src/app/[locale]/guides/permission-sets/en.mdx

```diff
@@ -0,0 +1,195 @@
+import {Container} from "@/components/Container"
+import clearsky from "./clearsky.png"
+import skyblur from "./skyblur-oauth.png"
+import granular from "./skyblur-granular.png"
+import lexicon from "./skyblur-lexicon.png"
+
+export const metadata = {
+  title: 'Permission Set Guide',
```

Comment:
can we clarify the scope more at the title/link level?

"Permission Requests", description: "A guide for app developers"

the sets part ("permission sets") is a mechanism folks might know they need to be aware off